products/ros2.md

```diff
@@ -0,0 +1,83 @@
+---
+title: ROS2
+category: os
+iconSlug: ros
+permalink: /ros2
+releasePolicyLink: https://docs.ros.org/en/rolling/Releases.html
+activeSupportColumn: false
+releaseColumn: true
+eolColumn: End Of Life
+versionCommand: printenv | grep -i ROS
+releaseDateColumn: true
+releaseLabel: '__CODENAME__'
+changelogTemplate: 'http://docs.ros.org/en/__RELEASE_CYCLE__/index.html'
+releases:
+-   releaseCycle: 'humble'
+    codename: 'Humble Hawksbill'
+    eol: 2027-05-01
+    releaseDate: 2022-05-23
+    latestReleaseDate: 2022-05-23
+    latest: humble
+-   releaseCycle: 'galactic'
+    codename: 'Galactic Geochelone'
+    eol: 2022-12-09
+    releaseDate: 2021-05-23
+    latestReleaseDate: 2021-05-23
+    latest: galactic
+-   releaseCycle: 'foxy'
+    codename: 'Foxy Fitzroy'
+    eol: 2023-05-01
+    releaseDate: 2020-06-05
+    latestReleaseDate: 2020-06-05
+    latest: foxy
+-   releaseCycle: 'eloquent'
+    codename: 'Eloquent Elusor'
+    eol: 2020-11-01
+    releaseDate: 2019-11-22
+    latestReleaseDate: 2019-11-22
+    latest: 'eloquent'
+-   releaseCycle: 'dashing'
+    codename: 'Dashing Diademata'
+    eol: 2021-05-01
+    releaseDate: 2019-05-31
+    latestReleaseDate: 2019-05-31
+    latest: 'dashing'
+-   releaseCycle: 'crystal'
+    codename: 'Crystal Clemmys'
+    eol: 2019-12-01
+    releaseDate: 2018-12-14
+    latestReleaseDate: 2018-12-14
+    latest: 'crystal'
+-   releaseCycle: 'bouncy'
+    codename: 'Bouncy Bolson'
+    eol: 2019-07-01
+    releaseDate: 2018-07-02
+    latestReleaseDate: 2018-07-02
+    latest: 'bouncy'
+-   releaseCycle: 'ardent'
+    codename: 'Ardent Apalone'
+    eol: 2018-12-01
+    releaseDate: 2017-12-08
+    latestReleaseDate: 2017-12-08
+    latest: 'ardent'
+
+
+---
+
+> ROS (Robot Operating System) provides libraries and tools to help software developers create robot applications.
```

Comment:
Should talk about ROS2, and maybe clarify about what's different.

Reply:
Added more description on ROS2.